fastddsspy_tool/src/cpp/tool/Controller.cpp

```diff
@@ -348,8 +418,13 @@ void Controller::print_command_(
     // Print topic
     else
     {
-        ddspipe::core::types::DdsTopic topic = participants::ModelParser::get_topic(*model_, arguments[1]);
-        if (topic.m_topic_name != arguments[1])
+        std::string topic_name = arguments[1];
+
+        ddspipe::core::types::WildcardDdsFilterTopic filter_topic;
+        filter_topic.topic_name = topic_name;
+
+        std::set<eprosima::ddspipe::core::types::DdsTopic> topics = participants::ModelParser::get_topics(*model_, filter_topic);
+        if (topics.empty())
         {
             view_.show_error(STR_ENTRY
```

Comment:
This comment might not make much sense with wildcards right?

Reply:
Updated